### PR TITLE
Release workflows for json parser and queue integration.

### LINKED ISSUE #6: Create a NUGET entry for each project
Create a nuget project feed for each of the libraries created: Orchestrator, RabbitMq and Parser.

### LINKED ISSUE #7: CD release build
Send the artifact with a new version to the Nuget

### PR DESCRIPTION
**Pull request by ChrisGawlinski**

### CODE REVIEW

File: .github/workflows/Parser-Json-Release-Build.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --no-restore
+      - name: Test
+        run: dotnet test --no-build --verbosity normal
+      - name: Pack Orchestrator
+        run: dotnet pack ${GITHUB_WORKSPACE}/Parsers/Prognetics.Saga.Parser.Json/Prognetics.Saga.Parser.Json.csproj -p:PackageVersion=${{ github.ref_name }}
+      - name: Publish to nuget.org
+        run: dotnet nuget push ${GITHUB_WORKSPACE}/Parsers/Prognetics.Saga.Parser.Json/bin/Debug/*.nupkg -k ${{ secrets.NUGETAPIKEY }} -s https://api.nuget.org/v3/index.json
```
Comment (SzymonWelter) [inline .github/workflows/Parser-Json-Release-Build.yml]: I would use the release configuration for building

**Review by SzymonWelter — CHANGES_REQUESTED** [review]

File: .github/workflows/Parser-Json-Release-Build.yml (hunk 1 above)
Comment (ChrisGawlinski) [reply .github/workflows/Parser-Json-Release-Build.yml]: Updated! Good catch!

File: .github/workflows/Parser-Json-Release-Build.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --no-restore
+      - name: Test
+        run: dotnet test --no-build --verbosity normal
+      - name: Pack Orchestrator
+        run: dotnet pack ${GITHUB_WORKSPACE}/Parsers/Prognetics.Saga.Parser.Json/Prognetics.Saga.Parser.Json.csproj -p:PackageVersion=${{ github.ref_name }} -c Release
+      - name: Publish to nuget.org
+        run: dotnet nuget push ${GITHUB_WORKSPACE}/Parsers/Prognetics.Saga.Parser.Json/bin/Debug/*.nupkg -k ${{ secrets.NUGETAPIKEY }} -s https://api.nuget.org/v3/index.json
```
Comment (SzymonWelter) [inline .github/workflows/Parser-Json-Release-Build.yml]: You're still pushing nuget from the Debug directory

File: .github/workflows/QueueIntegration-RabbitMq-Release-Build.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --no-restore
+      - name: Test
+        run: dotnet test --no-build --verbosity normal
+      - name: Pack Orchestrator
+        run: dotnet pack ${GITHUB_WORKSPACE}/QueueIntegrations/Prognetics.Saga.Queue.RabbitMQ/Prognetics.Saga.Queue.RabbitMQ.csproj -p:PackageVersion=${{ github.ref_name }} -c Release
+      - name: Publish to nuget.org
+        run: dotnet nuget push ${GITHUB_WORKSPACE}/QueueIntegrations/Prognetics.Saga.Queue.RabbitMQ/bin/Debug/*.nupkg -k ${{ secrets.NUGETAPIKEY }} -s https://api.nuget.org/v3/index.json
```
Comment (SzymonWelter) [inline .github/workflows/QueueIntegration-RabbitMq-Release-Build.yml]: Debug -> Release

**Review by SzymonWelter — CHANGES_REQUESTED** [review]

File: .github/workflows/Orchestrator-Release-Build.yml [hunk 4]
```diff
@@ -20,6 +20,6 @@ jobs:
       - name: Test
         run: dotnet test --no-build --verbosity normal
       - name: Pack Orchestrator
-        run: dotnet pack ${GITHUB_WORKSPACE}/Orchestrator/Prognetics.Saga.Orchestrator/Prognetics.Saga.Orchestrator.csproj -p:PackageVersion=${{ github.ref_name }}
+        run: dotnet pack ${GITHUB_WORKSPACE}/Orchestrator/Prognetics.Saga.Orchestrator/Prognetics.Saga.Orchestrator.csproj -p:PackageVersion=${{ github.ref_name }} -c Release
       - name: Publish to nuget.org
         run: dotnet nuget push ${GITHUB_WORKSPACE}/Orchestrator/Prognetics.Saga.Orchestrator/bin/Debug/*.nupkg -k ${{ secrets.NUGETAPIKEY }} -s https://api.nuget.org/v3/index.json
```
Comment (SzymonWelter) [inline .github/workflows/Orchestrator-Release-Build.yml]: Debug -> Release

File: .github/workflows/Parser-Json-Release-Build.yml (hunk 2 above)
Comment (ChrisGawlinski) [reply .github/workflows/Parser-Json-Release-Build.yml]: Changed!

File: .github/workflows/QueueIntegration-RabbitMq-Release-Build.yml (hunk 3 above)
Comment (ChrisGawlinski) [reply .github/workflows/QueueIntegration-RabbitMq-Release-Build.yml]: Changed!

File: .github/workflows/Orchestrator-Release-Build.yml (hunk 4 above)
Comment (ChrisGawlinski) [reply .github/workflows/Orchestrator-Release-Build.yml]: Changed!

**Review by SzymonWelter — APPROVED** [review]

**Pull request merged**